robot_modeling/DQ_DifferentialDriveRobot.m

```diff
@@ -34,7 +34,11 @@
 % DQ Robotics website: dqrobotics.github.io
 %
 % Contributors to this file:
-%     Bruno Vihena Adorno - adorno@ufmg.br
+%    1.  Bruno Vihena Adorno - adorno@ufmg.br
```

Comment:
@juanjqo, could you please update my email address? Please use adorno@ieee.org

Reply:
@bvadorno sure thing!